android/app/src/main/res/values/colors.xml

```diff
@@ -1,3 +1,3 @@
 <resources>
-    <color name="bootsplash_background">#0E1B31</color>
+    <color name="bootsplash_background">#050F20</color>
```

Comment:
Should we be setting colours like this globally, so they can be used by other platforms, e.g. tvOS?

Reply:
The colours for splashes do need to be defined on the native side for the splash screens, but we could call them the same on tvOS and Android and reference them in the app. The advantage being that native theme/contrast adjustments would then also apply in-app, automatically...may be worth considering. https://reactnative.dev/docs/platformcolor.